app/src/main/java/cellscanner/wowtor/github/com/cellscanner/recorder/Recorder.java

```diff
@@ -0,0 +1,27 @@
+package cellscanner.wowtor.github.com.cellscanner.recorder;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+
+
+/**
+ * Responsible for controlling the recording functionality
+ */
+public class Recorder {
+    // TODO: PICK A NAME NOT `MY_PREFS_NAME`
+    final private static String PREFS_NAME = "MY_PREFS_NAME";
```

Comment:
Should be application name? `App.TITLE`?

Reply:
OK